lib/routes/18comic/utils.ts

```diff
@@ -19,6 +23,72 @@ const getRootUrl = (domain) => {
     return `https://${domain}`;
 };
 
+const apiMapCategory = (category) => {
+    switch (category) {
+        case 'another':
+            return '其他漫畫';
+        case 'doujin':
+            return '同人';
+        case 'hanman':
+            return '韓漫';
+        case 'meiman':
+            return '美漫';
+        case 'short':
+            return '短篇';
+        case 'single':
+            return '單本';
+        default:
+            return null;
+    }
+};
+
+const getApiUrl = () => `https://${apiDomain}`;
+
+// using api to fetch data
+const ProcessApiItems = async (apiUrl) => {
+    apiUrl = apiUrl.replace(/\?$/, '');
+    // get timestamp using javascript native api
+    const ts = Date.now();
+    const tokenParam = `${ts},1.7.5`;
+    // md5 from {token ts + "18comicAPP"
+    let token = `${ts}18comicAPP`;
+    token = md5(token);
+
+    const response = await got(apiUrl, {
+        headers: {
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36',
```

Comment:
Why use this UA? Does the site only works with this fixed UA string over RSSHub's [randomised UA](https://github.com/DIYgod/RSSHub/blob/ef22a71dd60ae8936204e26c3ed0ccce450e26a5/lib/config.ts#L464)?